src/fixup.js

```diff
@@ -158,6 +158,10 @@
       document.body.classList.add("outdated-spec");
       var node = document.createElement("p");
       node.classList.add("outdated-warning");
+      node.tabIndex = -1;
+      node.setAttribute("role", "dialog");
+      node.setAttribute("aria-modal", true);
+      node.setAttribute("aria-label", "outdatedWarning");
```

Comment:
This should be two words... or better, "This specification is outdated!". Imagine this is being read out by the screen reader in this context. Only add the label if you really need it, and there is no other element already giving this information inside the node.

Reply:
Good point. In order to reuse the title we already have in the node, I changed the attribute from `aria-label` to `aria-labelledby`